R/utils.R

```diff
@@ -5,15 +5,17 @@ NULL
 #' Concatenate multiple feature lists and add the sample id (origin of feature) as additional column.
 #' 
 #' @param features list List of tibbles containing extracted feature tables.
-concatenate_feature_tables <- function(features) {
-  for (i in seq_along(features)) {
-    if(!("sample_id" %in% colnames(features[[i]]))) {
-      features[[i]] <- tibble::add_column(features[[i]], sample_id = i)
+concatenate_feature_tables <- function(features, sample_names) {
+    if(!is.na(sample_names)) {
```

Comment:
`devtools::check()` doesn't like this. The problem is that `sample_names` might not be given and it's `NA` by default. However, if it __is__ given, it's a list and this condition produces a warning (although technically it works). 

Any idea how to properly solve this? Maybe change the default value of `sample_names` to an empty list and check for length here? I'm not sure how that would behave in general.

How to reproduce:
```R
> vec <- c("a", "b", "c")
> is.na(vec)
Warning message:
In if (is.na(vec)) { :
  the condition has length > 1 and only the first element will be used
```

Reply:
The simplest solution to omit the warning could be to use `anyNA` instead of `is.na`. This approach would work for the default value and will catch if any of the names are `NA` (which in our case means that there's something wrong with the names).

Reply:
Thanks! I just fixed it using `all`, but maybe your solution is even better. I have no idea honestly.

Reply:
Yes, you are right, but also, `sample_names` is quite a top-level parameter, so I'm not sure whether this is the right spot to check it.